packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -214,27 +210,21 @@ export default class ScriptTransformer {
             cwd: this._config.rootDir,
             exclude: [],
             extension: false,
-            // Needed for correct coverage as soon as we start storing a source map of the instrumented code
-            inputSourceMap: input.map,
             useInlineSourceMaps: false,
           },
         ],
       ],
-      /**
-       * It's necessary to be able to map back to original source from the instrumented code.
-       * The inline map is needed for debugging functionality, and exposing it as a separate file is needed
-       * for mapping stack traces. It's convenient to use 'both' here and avoid extracting the source map.
-       *
-       * Previous behavior of emitting no map when we can't map back to original source is preserved.
-       */
-      sourceMaps: canMapToInput ? 'both' : false,
```

Comment:
specifically this line that breaks. I can keep everything else and just remove this line (or set it to `false`) and everything works again. But I thought a full revert makes more sense than a partial one

Reply:
Thanks! Revert seems appropriate given how confusing this area is.

Hopefully can re-attempt this one soon.

Reply:
Sounds good, thanks!